test/issue207.js

```diff
@@ -0,0 +1,52 @@
+
+var tape = require('tape');
+var mjAPI = require("../lib/main.js");
+var jsdom = require('jsdom').jsdom;
+
+tape('Generate dummy speechText', function(t) {
+  t.plan(6);
+  mjAPI.start();
+  var input1 = 'x';
+  var input2 = '<math display="block" alttext="hello"><mi>x</mi></math>';
+  var expected = 'hello';
+  // basic text
+  mjAPI.typeset({
+    math: input1,
+    format: "TeX",
+    html: true,
+    svg: true,
+    mml: true
+  }, function(data) {
+    var document = jsdom(data.html).defaultView.document;
+    var element = document.querySelector('.MJXc-display');
+    var result = element.getAttribute('aria-label');
+    t.equal(result, input1, 'HTML output contains dummy speechText');
+    var document = jsdom(data.mml).defaultView.document;
+    var element = document.querySelector('math');
+    var result = element.getAttribute('alttext');
+    t.equal(result, input1, 'MathML output contains dummy speechText');
+    var document = jsdom(data.svg).defaultView.document;
+    var desc = document.querySelector('desc');
+    t.equal(result, desc.innerHTML, 'SVG output contains dummy speechText');
```

Comment:
Hm. I don't see how to refactor this without making the test worse. I don't want to give up on having differentiating messages from the tests. I also don't want to create some complicated function that generates fitting tests and messages dynamically because I worry that this will make the test less readable. 

Do you have any suggestions?